### PR TITLE
Fix for localstack container not starting on fips machines

### PR DESCRIPTION
**Pull request by fedinskiy**
**Check list**:

Your pull request:

- [ ] targets the `development` branch
- [ ] uses the `999-SNAPSHOT` version of Quarkus
- [x] has tests (`mvn clean test`)
- [x] works in native (`mvn clean package -Pnative`)
- [x] has native tests (`mvn clean verify -Pnative`)
- [ ] makes sure the associated guide must not be updated
- [ ] links the guide update pull request (if needed)
- [ ] updates or creates the `README.md` file (with build and run instructions)
- [ ] for new quickstart, is located in the directory _component-quickstart_
- [ ] for new quickstart, is added to the root `pom.xml` and `README.md`

Fixed https://github.com/quarkusio/quarkus-quickstarts/issues/1056

### CODE REVIEW

**Comment by pjgg** [comment]
@gsmet could you have a look?

**Comment by rsvoboda** [comment]
Guillaume is on PTO, I can review it.

But we need PR to development branch

File: amazon-sqs-quickstart/src/test/java/org/acme/sqs/SqsResource.java [hunk 1]
```diff
@@ -43,7 +43,8 @@
         }
 
         Map<String, String> properties = new HashMap<>();
-        properties.put("quarkus.sqs.endpoint-override", endpoint());
+        System.out.println(services.getEndpointOverride(Service.SQS));
```
Comment (rsvoboda) [inline amazon-sqs-quickstart/src/test/java/org/acme/sqs/SqsResource.java:46]: Remove that

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: amazon-sqs-quickstart/src/test/java/org/acme/sqs/SqsResource.java (hunk 1 above)
Comment (fedinskiy) [reply amazon-sqs-quickstart/src/test/java/org/acme/sqs/SqsResource.java]: Fixed

**Review by rsvoboda — APPROVED** [review]
Looks good, let's wait for CI. In the meantime please prepare PR for  development branch

**Comment by fedinskiy** [comment]
@rsvoboda development branch already has this fix(https://github.com/quarkusio/quarkus-quickstarts/commit/76b33720e056724d7d7b30da3d1cfcc03513ff3c), but it wasn't backported

**Comment by rsvoboda** [comment]
CI is green, merging

**Pull request merged**